qoffer-domain/src/main/groovy/life/qbic/portal/portlet/customers/create/CreateCustomer.groovy

```diff
@@ -1,13 +1,12 @@
 package life.qbic.portal.portlet.customers.create
 
 /**
- * <short description>
+ * This use case creates a customer in the system
```

Comment:
```suggestion
 * This use case creates a customer
```

Reply:
I added the 
> in the system
to show that this use case returns an object without logic to be stored inside the database/the qbic system somewhere